### DISCUSSION TITLE
Are factory sounds accessible on E2 Sampler hardware?

### DISCUSSION THREAD

**Discussion by genderbent** [bangcorrupt/hacktribe#100] [Q&A] [answered]
When installing Hacktribe on an E2S, is it still possible to access any of the factory soundbank, or does it need to be exported before installation, and then re-installed into the user banks starting at sample 501?

**Comment by bangcorrupt** [comment, marked as answer]
It needs to be exported before installation, and then re-installed into the user banks starting at sample 501.
